app/components/node/PoSFooter.tsx

```diff
@@ -50,7 +52,7 @@ class PoSFooter extends PureComponent<Props> {
           <Button
             style={{ marginLeft: '20px' }}
             onClick={action}
-            text={isLastMode ? 'CREATE DATA' : 'NEXT'}
+            text={nextLabel || (isLastMode ? 'CREATE DATA' : 'NEXT')}
```

Comment:
Move `isLastMode` logic outside the component, now you have an universal `nextLabel`:

```suggestion
            text={nextLabel || 'NEXT'}
```
+ replace `isLastMode` with `nextLabel="CREATE DATA"` inside `PoSSummary` component :-p